src/ewatercycle/forcing/_lisflood.py

```diff
@@ -112,23 +122,80 @@ def generate(  # type: ignore
         for var_name in var_names:
             variables[var_name]["end_year"] = endyear
 
+        # set crop to false to keep the entire globe (time consuming)
+        # because lisflood parameter set is global i.e.
+        # recipe.data["preprocessors"]["general"]["extract_shape"]["crop"] = False
+        # However, lisflood diagnostics line 144 gives error
+        # ValueError: The 'longitude' DimCoord points array must be strictly monotonic.
+
         # generate forcing data and retrieve useful information
         recipe_output = recipe.run()
         directory, forcing_files = data_files_from_recipe_output(recipe_output)
 
-        # TODO run lisvap
-        # TODO forcing_files['e0'] = ...
-
-        # instantiate forcing object based on generated data
         if run_lisvap:
-            raise NotImplementedError("Dont know how to run LISVAP.")
+            # Get lisvap specific options and make paths absolute
+            lisvap_config = str(to_absolute_path(run_lisvap["lisvap_config"]))
+            mask_map = str(to_absolute_path(run_lisvap["mask_map"]))
+            version = run_lisvap["version"]
+            parameterset_dir = str(to_absolute_path(run_lisvap["parameterset_dir"]))
+
+            # Reindex data because recipe cropped the data
+            # Also, create a sub dir for global dataset because xarray does not
```

Comment:
Naming the sub dir `global` is because of our usecase. Can we pick a more widely usable name like `reindexed` 
 or `uncropped`?

Reply:
Because the lisflood parameter set is called `lisflood_global-masked_01degree`. But now I think that "global" might not fit a different parameter-set in the future. So, perhaps `reindexed` is a better name because those files are the output of `reindex` function!

Reply:
Done!